src/monster.cpp

```diff
@@ -385,6 +385,33 @@ void monster::get_Attitude(nc_color &color, std::string &text) const
     }
 }
 
+std::pair<std::string, nc_color> hp_description( int cur_hp, int max_hp )
+{
+    std::string damage_info;
+    nc_color col;
+    if( cur_hp >= max_hp ) {
+        damage_info = _("It is uninjured");
```

Comment:
Nit: no dots at end of sentence looks OK in non-extended view, but not so well in extended.